docs/General/Presentations/tools.md

```diff
@@ -1,10 +1,38 @@
 # Presentation Tools
 
-TODO: Add descriptions of presentation tools
-
-- Google Sheets
-- Templates
-- Miro
-- Flaticon
-- Biorender
-- Slido
+This page describes standards, tools, and resources for creating presentations for the lab.
+
+**Google Sheets**
+
+All external presentations (conferences, workshops, external working groups), should be created on Google Sheets directly by or shared to the `bhklab.research@gmail.com` account.
+
+Internal presentations, such as journal clubs, should also be created on Google Sheets. *Link Lab Meeting and Journal Club pages here.
+
+
+**Templates**
+
+Several slide templates have already been developed for the lab: 
+
+- [BHKLab Internal Presentations Template](https://docs.google.com/presentation/d/1GRdR3ExBVbXwgmjkHBp82i3x_mdQZPzJNHsEz1LiYWk/edit)
+- [BHKLab External Presentations Template](https://docs.google.com/presentation/d/1B-iTfzHvVSOD9fW80NZz14ct6HENHWOgTxU8C673TmY/edit)
+
+
+
+**Miro**
+
+[Miro](https://miro.com/login/) is a collaborative white board platform that enables simple diagram and multi-figure panel creation for presentations and/or manuscripts. For diagrams that will be used and updated by others in the lab, please request a Miro Board be created via the BHK Lab Miro account.
+
+
+**Images and Icons**
+
+The following are open-source biological imaging databases and platforms. The BHK Lab holds a premium Flaticon account (*do I provide login details?*). If Biorender and/or BioArt are used, please remember to add them to the acknowledgements.
```

Comment:
I think the site is technically public so probably don't include login details. I would put something like "Contact Sisira/whoever for login details", etc.